### DISCUSSION TITLE
Downloading files with NestJS

### DISCUSSION THREAD

**Discussion by giscus** [notiz-dev/notiz#219] [General]
# Downloading files with NestJS

Setup type-safe endpoints for downloading files in your NestJS application.

https://notiz.dev/blog/type-safe-file-downloads

**Comment by LucasBBueno** [comment]
Thanks man, that's help me a lot :)

**Comment by abdou-1614** [comment]
thank you so much Sir! we are waiting new posts🙏

**Comment by kunhokimcareerday** [comment]
Thank you very much you saved me a day

**Comment by CaioRobs** [comment]
perfect!
